jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/Button.java

```diff
@@ -91,19 +43,4 @@ public boolean isDisabled() {
     public String getText() {
         return core().getText();
```

Comment:
Do you really need all methods in this class?

Reply:
this method uses in another material tests